src/main/java/org/broadinstitute/hellbender/tools/spark/sv/BreakpointEvidence.java

```diff
@@ -155,7 +184,53 @@ protected void serialize( final Kryo kryo, final Output output ) {
 
         @Override
         public String toString() {
-            return super.toString() + " Split " + cigar + (tagSA == null ? " SA: None" : (" SA: " + tagSA));
+            final StringBuffer out = new StringBuffer();
+            out.append(super.toString());
+            out.append("\t");
+            out.append("Split");
+            out.append("\t");
+            out.append(cigar);
+            out.append("\t");
+            out.append((tagSA == null ? " SA: None" : (" SA: " + tagSA)));
+            return out.toString();
+        }
+
+        @Override
+        public boolean hasDistalTargets() {
+            return tagSA != null;
+        }
+
+        @Override
+        public List<SVInterval> getDistalTargets(final ReadMetadata readMetadata) {
+            if (tagSA != null) {
+                final String[] saStrings = tagSA.split(";");
+                final List<SVInterval> supplementaryAlignments = new ArrayList<>(saStrings.length);
+                for (final String saString : saStrings) {
+                    SVInterval saInterval = saStringToSVInterval(readMetadata, saString);
+                    supplementaryAlignments.add(saInterval);
+                }
+                return supplementaryAlignments;
+            } else {
+                return null;
+            }
+        }
+
+        // todo: for now, taking the entire location of the supplementary alignment plus the uncertainty on each end
+        // A better solution might be to find the location of the actual clip on the other end of the reference,
+        // but that would be significantly more complex and possibly computationally expensive
+        private SVInterval saStringToSVInterval(final ReadMetadata readMetadata, final String saString) {
+            final String[] values = saString.split(",", -1);
+            if (values.length != 6) {
+                throw new GATKException("Could not parse SATag: "+ saString);
+            }
+            final String contigId = values[0];
+            final int pos = Integer.parseInt(values[1]);
+            final Cigar cigar = TextCigarCodec.decode(values[3]);
+
+            return new SVInterval( readMetadata.getContigID(contigId),
```

Comment:
Not that it matters a lot, but the pos in a SA tag as put by Bwa and the pos in `SVInterval` differ by 1, bwa ouput has 1-based system whereas `SVInterval` has 0-based.